src/components/Header/Header.scss

```diff
@@ -1,9 +1,150 @@
 @import '~style/variables';
 
-.wrapper {
-  border: 1px solid blue;
+header {
+  display: flex;
+  flex-direction: column;
+  background-color: $color-primary--500;
+  color: $color-white;
+
+  > * {
+    @include for-desktop-up {
+      padding-left: 30px;
+      padding-right: 30px;
+      max-height: 176px;
+    }
+  }
+
+  a {
+    color: white;
+    text-decoration: underline;
+    text-decoration-color: $color-highlight;
+  }
+}
+
+.govContainer {
+  display: flex;
+  height: 48px;
+  padding-left: 14px;
+  flex-flow: column wrap;
+  align-items: flex-start;
+  align-content: flex-start;
+  justify-content: center;
+
+  @include for-desktop-up {
+    padding-left: 30px;
+    height: initial;
+    align-items: baseline;
+    flex-flow: row nowrap;
+    justify-content: flex-start;
+    background-color: $color-primary--400;
+
+    > * {
+      margin-right: 8px;
+    }
+  }
+}
+
+.compactTitle,
+.compactSubtitle {
+  @include for-desktop-up {
+    display: none;
+  }
+}
+
+.govAu,
+.govWebsite {
+  display: none;
+
+  @include for-desktop-up {
+    display: initial;
+  }
+}
+
+.govAu {
+  font-size: 14px;
+  font-weight: 500;
+}
+
+.govWebsite {
+  font-size: 12px;
+  font-weight: 100;
+}
+
+.mainContainer {
+  display: none;
+
+  @include for-desktop-up {
+    display: flex;
+    flex-flow: column wrap;
+    justify-content: space-between;
+    margin-bottom: 24px;
+  }
+}
+
+.star {
+  @include for-desktop-up {
+    height: 18px;
+    width: 18px;
+    margin-top: 2px;
+    flex: 0 1 auto;
+  }
+
+  height: 22px;
+  width: 22px;
+  max-width: 22px;
+  flex: 0 0 100%;
+  margin-right: 5px;
+}
+
+.projectStatus {
+  margin-top: 24px;
+  size: 16px;
+  text-align: center;
+  width: 80px;
+  font-weight: 600;
+  border: 1px solid $color-badge;
+  border-radius: 17px;
+  background-color: $color-badge;
 }
 
 .title {
-  font-size: 20px;
+  font-size: 36px;
+  font-weight: 300;
+  line-height: 46px;
+  margin-top: 10px;
+  max-height: initial;
+}
+
+.compactTitle {
+  font-size: 14px;
+  font-weight: 600;
+  max-height: 16px;
+  flex: 0 0 16px;
```

Comment:
Minor, it took me a long while to work out that these are all actually in a wrapping column, so `16px` is really height here. To be more explicit (if you keep the whole flex-column setup) I think you can have `height: 16px` and `flex: none`.

I think `max-height` can go, no?